src/app/problems/problem-view/mcq-view-snippet/mcq-view-snippet.component.ts

```diff
@@ -43,7 +43,8 @@ export class McqViewSnippetComponent implements OnInit {
         if (!this.uqj.is_checkbox) {
             this.formData = this.formBuilder.group({
                 question: new FormControl(this.uqj.question.id),
-                solution: new FormControl(null, [Validators.required])
+                solution: new FormControl(null, [Validators.required]),
+                time_spent: new FormControl(parseInt(localStorage.getItem(this.uqj.id.toString())))
```

Comment:
I think this was supposed to be removed from the `formData`.

Reply:
Done.